.github/workflows/windows.yaml

```diff
@@ -0,0 +1,44 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+  pull_request:
+
+name: Windows
+
+jobs:
+  R-CMD-check:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: windows-latest, r: 'devel'}
+          - {os: windows-latest, r: 'release'}
+          - {os: windows-latest, r: 'oldrel'}
```

Comment:
I am not sure how much I want to bother with Windows (given that I already get it, with a slight delay, from r-universe). I feel I almost certainly do not want three versions.

But I can deal with that in a follow-up commit.

Reply:
Way slower than macOS + Linux too :-/

Reply:
I set it to r-release.  May dial it down to pull_requests. Thanks though.